content/millennium/dstu2/individuals/practitioner.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Practitioner | DSTU 2 API
+---
+
+# Practitioner
+
+* TOC
+{:toc}
+
+## Overview
+
+The Practitioner Resource provides information about a person formally involved in the care of a patient on behalf of a healthcare facility. Practitioners include and are not limited to physicians, nurses, pharmacists, therapists, technologists, and social workers.
+
+The following fields are returned if valued:
+
+  * [Practitioner id](http://hl7.org/fhir/DSTU2/resource-definitions.html#Resource.id){:target="_blank"}
+  * [Identifiers/Aliases such as NPI and DEA](http://hl7.org/fhir/DSTU2/practitioner-definitions.html#Practitioner.identifier){:target="_blank"}
+  * [Active (true/false)](http://hl7.org/fhir/DSTU2/practitioner-definitions.html#Practitioner.active){:target="_blank"}
+  * [Name](http://hl7.org/fhir/DSTU2/practitioner-definitions.html#Practitioner.name){:target="_blank"}
+  * [Contact (secure email and phone)](http://hl7.org/fhir/DSTU2/practitioner-definitions.html#Practitioner.telecom){:target="_blank"}
+  * [Address](http://hl7.org/fhir/DSTU2/practitioner-definitions.html#Practitioner.address){:target="_blank"}
+  * [Gender](http://hl7.org/fhir/DSTU2/practitioner-definitions.html#Practitioner.gender){:target="_blank"}
+  * [Date of birth](http://hl7.org/fhir/DSTU2/practitioner-definitions.html#Practitioner.birthDate){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:practitioner, :dstu2) %>
+
+## Search
+
+Search for Practitioners that meet supplied query parameters:
+
+    GET /Practitioner?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name         | Required?                                         | Type       | Description
+--------------|---------------------------------------------------|------------|-------------------------------------------------------------------------------------
```

Comment:
The Required and Type columns can be shrunk and the trailing dashes over Description can be shortened